src/hooks/useNotifications.tsx

```diff
@@ -82,24 +97,41 @@ const notificationQuery = gql`
               }
             }
           }
+          ... on PrivatePostNotification {
+            post {
+              id
+              __typename
+              ... on ActivePost {
+                authorId
+              }
+            }
+          }
         }
       }
     }
   }
 `;
 
-const supportedNotificationTypes = [
+export type SupportedNotifications = [
   'PostReplyNotification',
   'CircleAdminPostNotification',
   'SurveyAssignedNotification',
+  'PrivatePostNotification',
 ];
+
+function isSupportedNotification(
+  __typename: any,
+): __typename is SupportedNotifications {
+  return true;
+}
```

Comment:
Is this right? We want it to always be true?

Reply:
This is a mistake, I will update it.

Reply:
Updated - fairly certain what I have now is correct - let me know if you disagree